### PR TITLE
Send news tematica per domain

### PR DESCRIPTION
**Pull request by d2bit**
[task](https://trello.com/c/GSWdwvox/472-8-r-opcion-de-poder-filtrar-las-newsletter-por-dominios)
[suscribir related PR](https://github.com/Soluciones/suscribir/pull/44)

### CODE REVIEW

File: app/views/news_tematica/news_tematicas/edit.html.haml [hunk 1]
```diff
@@ -11,6 +11,11 @@
       %td.flota_der= link_to('Seleccionar contenidos', elegir_contenidos_news_tematica_path(@news_tematica), title: 'Permite seleccionar los titulares y mensajes que van a ser incluidos en la newsletter, quitando los duplicados y/o que no interesen.', style: 'margin-right: 10px')
     = campo f, 'html', tipo: 'area', autofocus: true, tr_style: 'vertical-align: top'
     = campo f, 'fecha_envio', tipo: 'datetime', label: 'Fecha&nbsp;envío', tr_style: 'height: 40px'
+    - if NewsTematica::Personalizacion.dominios.present?
+      %tr
+        %th
+          = f.label :dominio_de_envio, 'Dominio de envio:'
+        %td= f.collection_select :dominio_de_envio, NewsTematica::Personalizacion.dominios, :to_s, :to_s, include_blank: 'Todas'
```
Comment (Fernan2) [inline app/views/news_tematica/news_tematicas/edit.html.haml]: Dominios -> `include_blank: 'Todos'`, not Todas

**Comment by Fernan2** [comment]
Nice factories!
+1 when you change include_blank

**Comment by gponsu** [comment]
:+1:

**Comment by 007lva** [comment]
:+1:

**Comment by scruti** [comment]
:+1:

**Pull request merged**